superset-frontend/webpack.config.js

```diff
@@ -289,9 +289,10 @@ const config = {
         APP_DIR,
         './node_modules/@superset-ui/chart-controls',
       ),
+      react: path.resolve('./node_modules/react'),
```

Comment:
Perhaps we should have react as peer dep in superset-ui then?

Reply:
Hmm, that's a very good point. Let me try that.

Reply:
It apparently already is:
![image](https://user-images.githubusercontent.com/33317356/135883184-a4a4b1ab-d3e3-4e30-a5b2-62c241e616e7.png)

Reply:
Huh, then it's rather strange that it requires resolving